README.md

```diff
@@ -86,14 +86,26 @@ Mingle and talk to as many people as you can, all through the conference.
 *Could use some help here guys..*
 
 ### Visit
- - 2066 Crist Drive, Los Altos, California
+ - 2066 Crist Drive, Los Altos, California (?)
```

Comment:
It’s actually the address where it all started. The address is of the Garage where they built Apple 1. Intentionally left un narrated for people to be become curies and google it. Or has the satisfactions of knowing it already.